app.js

```diff
@@ -0,0 +1,33 @@
+const { response } = require('express');
+const express = require('express');
+const { homedir } = require('os');
+const app = express(); /* To initialize express - backend and giving all those 
+functionalities to 'app'*/
+
+app.use(express.static('public'))
+
+
+app.get('/home', (req, res, next) => {
+    
+    res.sendFile(__dirname + '/public/views/home.html')
+    
+}); 
+
+app.get('/about', (req, res, next) => {
+    console.log(req);
```

Comment:
Here it's not advisable to leave console.log() of a request / response, that can fill out your terminal really quick and if you get an error it will be harder to find it